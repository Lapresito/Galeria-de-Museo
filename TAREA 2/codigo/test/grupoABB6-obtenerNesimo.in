crearGrupoABB
crearVisitante 15 Rufino Garcia 21
agregarAGrupoABB
obtenerNesimoVisitanteGrupoABB 1
crearVisitante 3 Zafiro Fernandez 25
agregarAGrupoABB
obtenerNesimoVisitanteGrupoABB 1
obtenerNesimoVisitanteGrupoABB 2
crearVisitante 25 Casimira Sanchez 26
agregarAGrupoABB
crearVisitante 22 Zafiro Sanchez 33
agregarAGrupoABB
crearVisitante 20 Rufino Gonzalez 23
agregarAGrupoABB
crearVisitante 19 Esmeralda Ruiz 32
agregarAGrupoABB
crearVisitante 21 Esmeralda Garcia 58
agregarAGrupoABB
crearVisitante 2 Rufino Lopez 28
agregarAGrupoABB
crearVisitante 1 Valquiria Ruiz 27
agregarAGrupoABB
crearVisitante 6 Zafiro Rodriguez 42
agregarAGrupoABB
obtenerNesimoVisitanteGrupoABB 10
obtenerNesimoVisitanteGrupoABB 9
obtenerNesimoVisitanteGrupoABB 8
obtenerNesimoVisitanteGrupoABB 7
obtenerNesimoVisitanteGrupoABB 6
obtenerNesimoVisitanteGrupoABB 1
obtenerNesimoVisitanteGrupoABB 2
obtenerNesimoVisitanteGrupoABB 3
obtenerNesimoVisitanteGrupoABB 4
obtenerNesimoVisitanteGrupoABB 5
liberarGrupoABB
Fin
