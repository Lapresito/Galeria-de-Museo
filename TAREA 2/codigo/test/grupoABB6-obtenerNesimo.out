Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> Visitante creadox de forma exitosa.
3> Se ha agregado el visitante al grupo de forma exitosa.
4> Visitante nro 1 del grupoABB:
Visitante Rufino Garcia
Id: 15
Edad: 21
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> Visitante nro 1 del grupoABB:
Visitante Zafiro Fernandez
Id: 3
Edad: 25
8> Visitante nro 2 del grupoABB:
Visitante Rufino Garcia
Id: 15
Edad: 21
9> Visitante creadox de forma exitosa.
10> Se ha agregado el visitante al grupo de forma exitosa.
11> Visitante creadox de forma exitosa.
12> Se ha agregado el visitante al grupo de forma exitosa.
13> Visitante creadox de forma exitosa.
14> Se ha agregado el visitante al grupo de forma exitosa.
15> Visitante creadox de forma exitosa.
16> Se ha agregado el visitante al grupo de forma exitosa.
17> Visitante creadox de forma exitosa.
18> Se ha agregado el visitante al grupo de forma exitosa.
19> Visitante creadox de forma exitosa.
20> Se ha agregado el visitante al grupo de forma exitosa.
21> Visitante creadox de forma exitosa.
22> Se ha agregado el visitante al grupo de forma exitosa.
23> Visitante creadox de forma exitosa.
24> Se ha agregado el visitante al grupo de forma exitosa.
25> Visitante nro 10 del grupoABB:
Visitante Casimira Sanchez
Id: 25
Edad: 26
26> Visitante nro 9 del grupoABB:
Visitante Zafiro Sanchez
Id: 22
Edad: 33
27> Visitante nro 8 del grupoABB:
Visitante Esmeralda Garcia
Id: 21
Edad: 58
28> Visitante nro 7 del grupoABB:
Visitante Rufino Gonzalez
Id: 20
Edad: 23
29> Visitante nro 6 del grupoABB:
Visitante Esmeralda Ruiz
Id: 19
Edad: 32
30> Visitante nro 1 del grupoABB:
Visitante Valquiria Ruiz
Id: 1
Edad: 27
31> Visitante nro 2 del grupoABB:
Visitante Rufino Lopez
Id: 2
Edad: 28
32> Visitante nro 3 del grupoABB:
Visitante Zafiro Fernandez
Id: 3
Edad: 25
33> Visitante nro 4 del grupoABB:
Visitante Zafiro Rodriguez
Id: 6
Edad: 42
34> Visitante nro 5 del grupoABB:
Visitante Rufino Garcia
Id: 15
Edad: 21
35> Grupo liberado con exito.
36> Fin.
